src/PackageCompilerX.jl

```diff
@@ -17,8 +17,22 @@ current_process_sysimage_path() = unsafe_string(Base.JLOptions().image_file)
 all_stdlibs() = readdir(Sys.STDLIB)
 
 yesno(b::Bool) = b ? "yes" : "no"
-bitflag() = Int == Int32 ? `-m32` : `-m64`
-march() = (Int == Int32 ? `-march=pentium4` : ``)
+
+function bitflag()
+    if Sys.ARCH == :aarch64 || Sys.ARCH == :arm
+        return ``
+    else
+        return Int == Int32 ? `-m32` : `-m64`
+    end
+end
+
+function march()
+    if Sys.ARCH == :aarch64 || Sys.ARCH == :arm
+        return (Int == Int32 ? `-march=armv7-a` : `-march=armv8-a+crypto+simd`)
```

Comment:
Where did the `crypto+simd` come from?

Reply:
I think the `simd` is redundant since `crypto` seems to include `simd`

Reply:
I think 

It comes from @DilumAluthge 's comment

https://julialang.slack.com/archives/CB8A1RNCE/p1581112984235200

> Where did the crypto+simd come from?

Reply:
Okay, it doesn't seem Julia uses that by default but maybe it should!? 

https://github.com/JuliaCI/julia-buildbot/blob/489ad6dee5f1e8f2ad341397dc15bb4fce436b26/master/inventory.py#L156

Reply:
> maybe it should!?

- I'm not sure, I always use `-march=armv8` option to create Julia binary that runs on Jetson nano device.
- I think setting `-march=armv8` is better for me which is same as JuliaCI adopts, just in case.

Reply:
sorry 

`armv8-a` is correct not `armv8`